src/main/webapp/app/exam/participate/exam-participation.component.ts

```diff
@@ -286,7 +286,7 @@ export class ExamParticipationComponent implements OnInit, OnDestroy, ComponentC
             // if this was calculated to true by the server, we can be sure the student exam has finished
             return true;
         }
-        if (this.handInEarly || this.studentExam.submitted) {
+        if (this.handInEarly || (this.studentExam && this.studentExam.submitted)) {
```

Comment:
Would this lead to the same result? I am unsure.
```suggestion
        if (this.handInEarly || this.studentExam?.submitted) {
```

Reply:
Tested it and it prevents the NPE.